### ISSUE TITLE
hw: build error with gcc 10

### ISSUE THREAD

**Issue by cproc** [genodelabs/genode#4115]
When building `LIB=core-hw-imx53_qsb_tz` with gcc 10, the following error occurs:

```
  Library core-hw-imx53_qsb_tz
    COMPILE  spec/arm_v7/trustzone/kernel/vm.o
/.../repos/base-hw/src/core/spec/arm_v7/trustzone/kernel/vm.cc: In member function ‘virtual void Kernel::Vm::exception(Kernel::Cpu&)’:
/.../repos/base-hw/src/core/spec/arm_v7/trustzone/kernel/vm.cc:43:48: error: ‘fallthrough’ attribute ignored [-Werror=attributes]
   43 |  case Genode::Cpu_state::FAST_INTERRUPT_REQUEST:
      |                                                ^
```

**Comment by cproc** [comment]
Fixed in 543971a

**Comment by nfeske** [comment]
Merged to staging.
